Sources/VLCUI/VLCVideoPlayer/Proxy.swift

```diff
@@ -152,5 +152,42 @@ public extension VLCVideoPlayer {
         public func playNewMedia(_ newConfiguration: Configuration) {
             videoPlayerView?.setupVLCMediaPlayer(with: newConfiguration)
         }
+
+        /// Starts the recording process
+        /// - Parameter directoryPath: The directory path where the recording will be saved
+        /// - Returns: Indicates whether the recording started successfully
+        public func startRecording(directoryPath: String) -> Bool {
+            guard let mediaPlayer else { return false }
+            guard mediaPlayer.isPlaying else { return false }
+            mediaPlayer.startRecording(atPath: directoryPath)
+            return true
+        }
+        
+        /// Stops the recording process
+        /// - Returns: Indicates whether the recording was stopped successfully
+        public func stopRecording() {
+            mediaPlayer?.stopRecording()
+        }
+
+        /// Set the video aspect ratio
```

Comment:
We should also specify that invalid values would be ignored. I would prefer if we made something strongly typed, but dealing with the formatting from a `Double` to a string value would be annoying.